### DISCUSSION TITLE
How to render a transparent plane that receives shadows from objects above but also blocks shadowing receiving for objects below?

### DISCUSSION THREAD

**Discussion by Ankbzpx** [google/filament#4767] [Q&A]
Suppose a scene with two planes A and B, where both are [transparent shadow plane](https://github.com/google/filament/blob/main/ios/samples/hello-ar/Materials/shadow_plane.mat) and A is placed right above B. When placing an object above A, its shadow will be received by both A and B. Is there a method to prevent B from receiving its shadow?

**Comment by romainguy** [comment]
What you are describing isn't supported (and would be non trivial/expensive to implement).
